_data/projects/computer-build-calculator.yml

```diff
@@ -0,0 +1,10 @@
+name: Computer Build Calculator
+desc: A tool that calculates the best value PC components for a set price
+site: https://github.com/Computer-build-calculator
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- php
+- web
+upforgrabs:
+  name: help wanted
+  link: https://github.com/Computer-build-calculator
```

Comment:
This link should be a URL to a list of curated tasks that contributors can work on.